src/demos/calcite-loader.html

```diff
@@ -36,6 +36,14 @@ <h1>Calcite Loader</h1>
   <p style="text-align: center;">
     <calcite-loader is-active inline></calcite-loader>Inline Loader
   </p>
+      <calcite-loader is-active type="determinate" value="50" text="Determinate loader">
```

Comment:
Weird indentation here, but also I think you may be able to use the self-closing syntax on these bad boys?

Reply:
Ah, yeah missed formatting on the demo page. Will update.